packages/keystone/lib/List/index.js

```diff
@@ -219,7 +219,7 @@ module.exports = class List {
       listKey: key,
       access,
       defaultAccess: this.defaultAccess.list,
-    })['public'];
+    });
```

Comment:
This is the crux of the PR. We want to change `.access` to be something like `{ public: { read: true, ... } }`. All the other changes stem from this (and the corresponding change in the base field class).

For now we hard-code `schemaName = 'public'` in a bunch of places. These will be resolved in the next PR, which will make changes to the `Keystone` level APIs.

Reply:
should public be defaults to read: true and rest false? just to be more secure by default

Reply:
@gautamsi I believe that's coming in a future PR 👍

Reply:
Yep, the question of default access control is definitely on our minds at the moment and will be addressed once we have these mechanisms in place.